xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -669,12 +670,6 @@ void CRenderManager::CreateRenderer()
       m_pRenderer = new CRendererMediaCodec;
 #endif
     }
-    if (m_format == RENDER_FMT_MMAL)
```

Comment:
what is the purpose of RENDER_FMT_MMAL? can we drop it?
if MMAL is similar to DXVA that serves hw and sw decoding, i would leave it here as well.